build.gradle

```diff
@@ -44,14 +44,27 @@ subprojects {
         }
     }
 
+    //Required by JitPack for source packing
+    apply plugin: 'maven'
+
+    task sourcesJar(type: Jar, dependsOn: classes) {
+        classifier = 'sources'
+        from sourceSets.main.allSource
+    }
+
+    artifacts {
+        archives sourcesJar
+    }
+
     dependencies {
         implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:${kotlin_version}"
         implementation "org.jetbrains.kotlin:kotlin-reflect:${kotlin_version}"
 
         compile 'com.github.alexeykoren:konfig:2.0.4'
 
         implementation 'io.github.microutils:kotlin-logging:1.6.22'
-        implementation 'io.reactivex:rxjava:1.3.8'
+        //Version should be same with one used by corda node.
+        implementation 'io.reactivex:rxjava:1.2.4'
```

Comment:
Let's use our common practice and specify version as separated variable

Reply:
I think that we had this discussion, and came to conclusion that only multiple time used libs should be added to this list.
I think that osesov and voddan were in this topic.

Reply:
Just to be clear, the last common conclusion on this was to specify versions for all libraries via global variables, and then specify all the versions in the root of the project. We haven't got to implementing it in a single project, but that's another point.

Reply:
Then i would suggest to create another PR and move all versions to specific place